src/cascadia/TerminalControl/TermControl.idl

```diff
@@ -77,6 +77,8 @@ namespace Microsoft.Terminal.TerminalControl
         event Windows.Foundation.TypedEventHandler<TermControl, Object> SetTaskbarProgress;
         event Windows.Foundation.TypedEventHandler<TermControl, NoticeEventArgs> RaiseNotice;
         event Windows.Foundation.TypedEventHandler<Object, Object> WarningBell;
+        event Windows.Foundation.TypedEventHandler<Object, Object> HideCursor;
+        event Windows.Foundation.TypedEventHandler<Object, Object> RestoreCursor;

```

Comment:
```suggestion
        event Windows.Foundation.TypedEventHandler<Object, Object> HideMouseCursor;
        event Windows.Foundation.TypedEventHandler<Object, Object> RestoreMouseCursor;
```
Maybe? Just to better distinguish the difference between the mouse cursor and terminal cursor

Reply:
Thanks! Done.